apps/codeforafrica/src/components/OpportunityCardList/OpportunityCardList.js

```diff
@@ -0,0 +1,24 @@
+import { Section } from "@commons-ui/core";
+import React from "react";
+
+import OpportunityCard from "@/codeforafrica/components/OpportunityCard";
+
+const OpportunityCardList = React.forwardRef(function OpportunityCardList(
+  props,
+  ref
+) {
+  const { opportunities } = props;
+
+  if (!opportunities?.length) {
+    return null;
+  }
+  return (
+    <Section sx={{ pb: "20px" }} ref={ref}>
```

Comment:
1. `px` is missing for `xs`.
2. The size for this section should be narrower than the default one for `md` and up.